packages/chrome/src/styled/header/StyledHeader.ts

```diff
@@ -5,21 +5,55 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import styled, { ThemeProps, DefaultTheme, css } from 'styled-components';
 import { retrieveComponentStyles, getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
 import { StyledLogoHeaderItem } from './StyledLogoHeaderItem';
 import { getNavItemHeight } from '../nav/StyledBaseNavItem';
 
 const COMPONENT_ID = 'chrome.header';
 
-export interface IStyledHeaderProps {
+export interface IStyledHeaderProps extends ThemeProps<DefaultTheme> {
   isStandalone?: boolean;
 }
 
-export const getHeaderHeight = (props: ThemeProps<DefaultTheme>) => {
-  return getNavItemHeight(props);
+const colorStyles = (props: IStyledHeaderProps) => {
+  const backgroundColor = props.theme.colors.background;
+  const borderColor = getColor(
+    'neutralHue',
+    props.theme.colors.base === 'dark' ? 700 : 300,
+    props.theme
+  );
+  const boxShadow = props.theme.shadows.lg(
+    '0',
+    `${props.theme.space.base * 2.5}px`,
+    getColor('chromeHue', 600, props.theme, 0.15)!
+  );
+  const foregroundColor = getColor('neutralHue', 600, props.theme);
+
+  return css`
+    border-bottom-color: ${borderColor};
+    box-shadow: ${props.isStandalone && boxShadow};
+    background-color: ${backgroundColor};
+    color: ${foregroundColor};
+  `;
+};
+
+const sizeStyles = (props: IStyledHeaderProps) => {
+  const border = props.theme.borders.sm;
+  const fontSize = props.theme.fontSizes.md;
+  const height = getNavItemHeight(props);
+  const padding = `${props.theme.space.base}px`;
+
+  return css`
+    border-bottom: ${border};
+    padding: 0 ${padding};
+    height: ${height};
+    font-size: ${fontSize};
+  `;
```

Comment:
This is an example of following the [documented](https://github.com/zendeskgarden/react-components/blob/main/docs/api.md#view-components) `colorStyles` and `sizeStyles` for view components. We'll expect to see this structure more fastidiously followed during dark mode development